### PR TITLE
L0 unit tests for the PRCA task implementation.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

**Comment by HSAR** [comment]
**Context**
The base branch "users/sschen/wip" is serving as the feature branch for this work. It has two feature commits which were reverted from master (it shouldn't have been in master in the first place but for a mistake on my part).

This commit contains unit tests that were previously in a different repository, along with metadata that allows them to be run using the vsts-tasks makefile.

File: Tasks/PullRequestCodeAnalysis/PRCA/PrcaOrchestrator.ts [hunk 1]
```diff
@@ -65,7 +67,7 @@ export class PrcaOrchestrator {
             throw new ReferenceError('token');
         }
 
-        let creds = web.getPersonalAccessTokenHandler(token);
+        let creds = web.getBearerHandler(token);
```
Comment (HSAR) [inline Tasks/PullRequestCodeAnalysis/PRCA/PrcaOrchestrator.ts]: Small API change. Tested as working.

File: Tasks/PullRequestCodeAnalysis/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Pull Request Code Analysis",
-  "loc.helpMarkDown": "[More Information](   NEEDS LINK    )",
+  "loc.helpMarkDown": "[More Information](https://aka.ms/prcabuildtask)",
```
Comment (HSAR) [inline Tasks/PullRequestCodeAnalysis/Strings/resources.resjson/en-US/resources.resjson]: Fixes 283070.

**Comment by bgavrilMS** [comment]
LGTM

File: Tasks/PullRequestCodeAnalysis/Tests/L0.ts [hunk 3]
```diff
@@ -0,0 +1,749 @@
...
+    expectedLine: number,
+    expectedPriority: number) {
+
+    chai.expect(actualMessage.content).to.equal(expectedContent, 'Content mismatch');
+    chai.expect(actualMessage.file).to.equal(expectedFile, 'File mismatch');
+    chai.expect(actualMessage.line).to.equal(expectedLine, 'Line mismatch');
+    chai.expect(actualMessage.priority).to.equal(expectedPriority, 'Priority mismatch');
+}
+
+describe('The PRCA', function () {
```
Comment (HSAR) [inline Tasks/PullRequestCodeAnalysis/Tests/L0.ts]: These are the tests. No functional changes were made.

File: Tasks/PullRequestCodeAnalysis/package.json [hunk 4]
```diff
@@ -11,10 +11,14 @@
...
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^2.5.3"
+  },
   "dependencies": {
     "fs": "0.0.2",
     "q": "1.4.1",
     "vso-node-api": "5.1.1",
-    "vsts-task-lib": "0.9.7"
+    "vsts-task-lib": "0.9.20"
```
Comment (HSAR) [inline Tasks/PullRequestCodeAnalysis/package.json]: This is required in order to use the new test framework classes and methods.

**Comment by bgavrilMS** [comment]
LGTM

**Pull request merged**